sample/src/main/java/com/github/zawadz88/materialpopupmenu/sample/DarkActivity.kt

```diff
@@ -316,6 +316,23 @@ class DarkActivity : AppCompatActivity() {
         popupMenu.show(this@DarkActivity, view)
     }
 
+    @OnClick(R.id.customPaddingTextView)
```

Comment:
Thanks for adding a sample as well!

Reply:
No problem, although I would suggest you to try and combine Dark/Light activities and their layouts. Right now adding new items to sample is kind of annoying since you have to add everything 2. Base abstract class with methods that ask for correct style where necessary could be enough. 

But that's just a small suggestion :)

Reply:
I agree with you. I mean, that's what I would normally do in a normal project.
However, I've noticed that the more complicated the sample app is the higher the likelihood that people won't follow what's going on (had some weird questions on some other projects on GitHub...). That's why I went for this awful duplication in the first place :/
It is easier, for complete beginners especially, to just look at the sample code and e.g. copy it to your project.